### ISSUE TITLE
[DISCUSSION] Change executable name from geos to something more apt

### ISSUE THREAD

**Issue by lizziel** [geoschem/GCHPctm#49]
The GEOS-Chem executable has been called geos for a very long time. However, GEOS-Chem is not GEOS. I propose that we rename it to geoschem, or gchp, or something else that is not geos. Thoughts?

**Comment by sdeastham** [comment]
Agreed. `gchp` makes sense to me as a name, and perhaps `gcclassic` for GC-Classic (obviously)?
